cmd/cli/namespace_list.go

```diff
@@ -77,10 +77,11 @@ func (l *namespaceListCmd) run() error {
 	}
 
 	w := newTabWriter(l.out)
-	fmt.Fprintln(w, "NAMESPACE\tMESH\t")
+	fmt.Fprintln(w, "NAMESPACE\tMESH\tINJECTION\t")
 	for _, ns := range namespaces.Items {
 		osmName, _ := ns.ObjectMeta.Labels[constants.OSMKubeResourceMonitorAnnotation]
-		fmt.Fprintf(w, "%s\t%s\t\n", ns.Name, osmName)
+		enabledSidecarInjection := ns.ObjectMeta.Annotations[constants.SidecarInjectionAnnotation]
```

Comment:
The annotation could be missing, in which case this will add a new key to the annotation. 
```suggestion
		sidecarInjectionEnabled , ok := ns.ObjectMeta.Annotations[constants.SidecarInjectionAnnotation]
		if !ok {
		    sidecarInjectionEnabled  = "-"  // not set
		 }
		
```